docs/OV_Runtime_UG/supported_plugins/GPU_RemoteTensor_API.md

```diff
@@ -125,17 +160,17 @@ Refer to the sections below to see pseudo-code of usage examples.
 
 This example uses the OpenCL context obtained from an executable network object.
 
-@snippet snippets/GPU_RemoteBlob_API0.cpp part0
+@snippet snippets/GPU_RemoteTensor_API0.cpp part0
 
 ### Running GPU Plugin Inference within User-Supplied Shared Context
 
-@snippet snippets/GPU_RemoteBlob_API1.cpp part1
+@snippet snippets/GPU_RemoteTensor_API1.cpp part1
 
 ### Direct Consuming of the NV12 VAAPI Video Decoder Surface on Linux
 
-@snippet snippets/GPU_RemoteBlob_API2.cpp part2
+@snippet snippets/GPU_RemoteTensor_API2.cpp part2
```

Comment:
BTW, such big snippets eat a lot of screen - how do you think, can we hide them under a spoiler? See https://github.com/openvinotoolkit/openvino/blob/releases/2021/4/docs/IE_DG/Integrate_with_customer_application_new_API.md#step-2-optional-configure-input-and-output-of-the-model `<div class="collapsible-section">`

Reply:
Done